### PR TITLE
Background arcs

### PR DESCRIPTION
**Pull request by shsteimer**
Test URLs:
- After: https://background-arcs--ups--aemsites.aem.page

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -376,6 +379,41 @@ main .section.dark {
   background-color: darkgray;
 }
 
+main .section.background-brown {
```
Comment (shsteimer) [inline styles/styles.css:382]: I suspect with some effort, the svg's could be avoided entirely and just use `clip-path: path(...)` but I'm not sure if it's worth the effort.

Comment (shsteimer) [reply styles/styles.css:382]: naming is hard

I settled on `background-arc` and `background-brown` based largely on how the current classes are named, but don't love that either.

let me know if you have thoughts/opinion

File: blocks/footer/footer.css [hunk 2]
```diff
@@ -15,7 +15,7 @@ footer {
 }
 
 footer .footer {
-  max-width: 1400px;
+  max-width: var(--max-content-width);
```
Comment (fkakatie) [inline blocks/footer/footer.css:18]: good call.

File: styles/styles.css (hunk 1 above)
Comment (fkakatie) [reply styles/styles.css:382]: AGREED. I think those are totally sufficient for now.

**Review by fkakatie — APPROVED** [review]

**Pull request merged**